volume-calculator/main.js

```diff
@@ -0,0 +1,30 @@
+////// Notes //////
+
+// -Need to create a function.
+// -Function name will be "volumeCalculator".
+// -The function will receive three different box measurements, and the goal
+//  is to find the volume of each box, and add the box volumes together to
+//  get the total volume.
+// -Will need to get each box volume stored in their own storage after finding volume.
+// Example input/output: volumeCalculator([[3, 3, 3], [4, 4, 4], [5, 5, 5]]) -> 216
+
+////// Pseudo Code //////
+
+// -Create a function named "volumeCalculator".
+// -Within the function, create a loop that looks through each set of box measurements.
+//   -Within the loop, create three storage containers for each box measurement after the volume
+//    has been found.
+// -Return the result of multiplying box1, box2, and box3 together.
+
+function volumeCalculator(boxesArray) {
+  for (var i=0; i<boxesArray.length; i++) {
```

Comment:
Based on your solution of multiplying each box out individually, this doesn't require a loop, since you are doing all of the work for the function at once already in the loop code block 👍